docs/source/top_level_elements/physicalDescription.rst

```diff
@@ -0,0 +1,273 @@
+physicalDescription
+===================
+
+About
+-----
+
+This section covers the top-level element `<physicalDescription> <https://www.loc.gov/standards/mods/userguide/physicaldescription.html>`_ and all of its sub-elements:
+<form>, <reformattingQuality>, <internetMediaType>, <extent>, <digitalOrigin>, and <note>. Currently UTK
+does not have any existing values for <reformattingQuality>.
+
+The hope is for all records to at least have a value for mods:physicalDescription/mods:form, but this is not the case
+presently. 6,074 have no form term at all (collections affected include arrsimple, tdh, revwar, civilwar, insurancena,
+fbpro, and thompson).
+
+physicalDescription - Null values
+---------------------------------
+
+All 460 records associated with `Images: From Pi Beta Phi to Arrowmont <https://digital.lib.utk.edu/collections/islandora/object/arrowmont%3Aarrsimple>`_
+have empty <physicalDescription> nodes.
+
+Here's `arrsimple:105 <https://digital.lib.utk.edu/collections/islandora/object/arrsimple%3A105/datastream/MODS/view>`_ as an example:
+
+.. code-block:: xml
+
+    <physicalDescription/>
+
+physicalDescription.extent - Null values
+--------------------------------------
```

Comment:
Your underline is too short.

Reply:
@markpbaggett - thanks for catching this.